core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -97,7 +97,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
 
   val startTime = System.currentTimeMillis()
 
-  private[spark] val stopped: AtomicBoolean = new AtomicBoolean(false)
+  @DeveloperApi val stopped: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
can we expose a slightly nicer API? Right now this is an atomic boolean and the user has to do something like
`sc.stopped.get` which is super clunky. It would be better if they can do `sc.isStopped` or something